app/bundles/LeadBundle/Entity/LeadField.php

```diff
@@ -109,6 +109,8 @@ class LeadField extends FormEntity implements CacheInvalidateInterface
 
     private ?bool $isIndex = false;
 
+    private ?bool $isSearchable = false;
```

Comment:
I don't see a reason for nullable property here. It will always have a bool value when you define it as default here. Please update also the getter and setter.
```suggestion
    private bool $isSearchable = false;
```